internal/lib/lib.go

```diff
@@ -1319,3 +1319,11 @@ func GetK8sMinSupportedVersion() string {
 func GetK8sMaxSupportedVersion() string {
 	return k8sMaxVersion
 }
+
+func IsVCFCluster() bool {
+	vcfCluster := os.Getenv(VCF_CLUSTER)
```

Comment:
How will this value populated to AKO POD? will by default in VCF each node have this?